components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/introspection/IntrospectionResponseBuilder.java

```diff
@@ -78,6 +78,17 @@ public IntrospectionResponseBuilder setIssuedAt(long issuedAt) {
         return this;
     }
 
+    /**
```

Comment:
Lets add a description for the method and description for the return variable

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1984/commits/59ba3e10b73d02dcf0d68de230df5add71a2ab97